lib/bucky/test_equipment/test_case/abst_test_case.rb

```diff
@@ -8,6 +8,8 @@ module TestEquipment
     module TestCase
```

Comment:
It is not necessary to fix this file, maybe.

Reply:
You are totally right!
Thanks a lot.